crates/agent/src/publications/specs.rs

```diff
@@ -282,8 +284,8 @@ pub fn validate_transition(
 
         // Verify that the live specification has not existed and then been deleted in the past.
         // TODO(johnny): remove once we introduce data plane pet-names.
-        if live_type.is_none() && draft_type.is_some() && *last_pub_id != pub_id   {
```

Comment:
@jgraettinger so I'm pretty sure that this was actually wrong, and my rustfmt fixed it. The spacing was off, there shouldn't have been two spaces here or two indents below

Reply:
👍